near-sdk/src/lib.rs

```diff
@@ -99,10 +98,330 @@
+/// }
+///
+/// #[near]
+/// impl Contract {
+///    pub fn multiply_by_five(&mut self, number: u64) -> Promise {
+///        ext_calculator::ext(self.calculator_account.clone())
+///            .with_static_gas(Gas::from_tgas(5))
+///            .mult(number, 5)
+///    }
+/// }
+///
+/// ```
+///
+/// See more information about role of ext_contract in [NEAR documentation](https://docs.near.org/build/smart-contracts/anatomy/crosscontract)
+pub use near_sdk_macros::ext_contract;
+
+/// This macro is deprecated. Use [macro@near] instead. The difference between `#[near]` and `#[near_bindgen]` is that
+/// with `#[near_bindgen]` you have to manually add boilerplate code for structs and enums so that they become Json- and Borsh-serializable:
+/// ```rust
+/// use near_sdk::{near_bindgen, NearSchema, borsh::{BorshSerialize, BorshDeserialize}};
+///
+/// #[near_bindgen]
+/// #[derive(BorshSerialize, BorshDeserialize, NearSchema)]
+/// #[borsh(crate = "near_sdk::borsh")]
+/// struct MyStruct {
+///    pub name: String,
+/// }
+/// ```
+/// Instead of:
+/// ```rust
+/// use near_sdk::near;
+///
+/// #[near(serializers=[borsh])]
+/// struct MyStruct {
+///     pub name: String,
+/// }
+/// ```
+pub use near_sdk_macros::near_bindgen;
+
+/// `BorshStorageKey` generates implementation for [BorshIntoStorageKey](crate::private::BorshIntoStorageKey) trait.
+/// It allows the type to be passed as a unique prefix for persistent collections.
+/// The type should also implement or derive [BorshSerialize](borsh::BorshSerialize) trait.
+///
+/// More information about storage keys in [NEAR documentation](https://docs.near.org/build/smart-contracts/anatomy/storage)
+/// ## Example
+/// ```rust
+/// use near_sdk::{BorshStorageKey, collections::Vector, near};
+///
+/// #[near(serializers=[borsh])]
+/// #[derive(BorshStorageKey)]
+/// pub enum StorageKey {
+///     Messages,
+/// }
+///
+/// // Define the contract structure
+/// #[near(contract_state)]
+/// pub struct Contract {
+///     messages: Vector<String>
+/// }
+///
+/// // Define the default, which automatically initializes the contract
+/// impl Default for Contract {
+///     fn default() -> Self {
+///         Self {
+///             messages: Vector::new(StorageKey::Messages)
+///         }
+///     }
+/// }
+/// ```
+pub use near_sdk_macros::BorshStorageKey;
+
+/// `PanicOnDefault` generates implementation for `Default` trait that panics with the following
+/// message `The contract is not initialized` when `default()` is called.
+/// This is a helpful macro in case the contract is required to be initialized with either `init` or
+/// `init(rust_state)`
+///
+/// ## Example
+/// ```rust
+/// use near_sdk::{PanicOnDefault, near};
+///
+/// #[near(contract_state)]
+/// #[derive(PanicOnDefault)]
+/// pub struct Contract {
+///     pub name: String,
+/// }
+/// ```
+pub use near_sdk_macros::PanicOnDefault;
+
+/// NOTE: This is an internal implementation for `#[near_bindgen(events(standard = ...))]` attribute.
```

Comment:
here it's near_bindgen, on the next line it's `near` again

Reply:
I ask there to use another macro, as it's not for public use.